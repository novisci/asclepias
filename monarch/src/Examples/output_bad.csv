reason,concepts,facts,begin,end
"after","home",WasAfter,0,4
