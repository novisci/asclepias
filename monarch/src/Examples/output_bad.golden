"[[[0,4],{\"getConcepts\":[\"home\"],\"getFacts\":\"WasAfter\",\"getSource\":null}]]"